message.module

```diff
@@ -306,7 +307,6 @@ function template_preprocess_message(&$variables) {
   $variables['teaser'] = $variables['view_mode'] == 'teaser';
   $variables['message'] = $variables['elements']['#message'];
 
-  /** @var \Drupal\message\MessageInterface $message */
```

Comment:
It's needed for type hunting for php storm -- you can see it in core as
well.
On Apr 4, 2016 8:22 AM, "HelenaEksler" <notifications@github.com> wrote:

> In message.module
> <https://github.com/Gizra/message/pull/97#discussion_r58329216>:
>
> > @@ -306,7 +307,6 @@ function template_preprocess_message(&$variables) {
> >    $variables['teaser'] = $variables['view_mode'] == 'teaser';
> >    $variables['message'] = $variables['elements']['#message'];
> >
> > -  /** @var \Drupal\message\MessageInterface $message */
>
> I think it's pretty clear that this should be the Message object in
> there. There is no need in this comment.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Gizra/message/pull/97/files/29c79202f3d5bc1dcb28ab1d2bf405f87a7c0e97#r58329216>
>